other/deployment-replicas-higher-than-pdb/deployment-replicas-higher-than-pdb.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: deployment-replicas-higher-than-pdb
+  annotations:
+    policies.kyverno.io/title: Ensure Deployment Replicas Higher Than PodDisruptionBudget
+    policies.kyverno.io/category: Other
+    policies.kyverno.io/subject: PodDisruptionBudget, Deployment
+    kyverno.io/kyverno-version: 1.11.4
+    kyverno.io/kubernetes-version: "1.27"
+    policies.kyverno.io/description: >-
+      PodDisruptionBudget resources are useful to ensuring minimum availability is maintained at all times.
+      Introducing a PDB where there are already matching Pod controllers may pose a problem if the author
+      is unaware of the existing replica count. This policy ensures that the minAvailable value is not
+      greater not equal to the replica count of any matching existing Deployment. If other Pod controllers
+      should also be included in this check, additional rules may be added to the policy which match those
+      controllers.
+spec:
+  validationFailureAction: Audit
+  background: true
+  rules:
```

Comment:
Could it make sense to merge the two policies addressing PDBs? For a user it is easy to remove one of the two rules - if required. And they are quite similar in nature.

Reply:
We really try and keep policies separated as much as possible except in cases where there is such a high degree of relativity/coupling that one rule either doesn't make sense without the other or where users could potentially fall into a trap (due to them not knowing Kyverno well enough, or Kubernetes) if left to their own devices to find a complementary one. Given that these could be separate use cases I think it makes sense to separate them where the other advantage is a nice description for each.

Reply:
Makes sense! Thanks for the detailed response!